import_flow_events.py

```diff
@@ -89,7 +89,8 @@
       -- but redshift doesn't support that.
       flow_time BIGINT NOT NULL ENCODE lzo,
       flow_id VARCHAR(64) NOT NULL DISTKEY ENCODE lzo,
-      type VARCHAR(64) NOT NULL ENCODE lzo
+      type VARCHAR(64) NOT NULL ENCODE lzo,
+      origin DATE NOT NULL ENCODE lzo
```

Comment:
Nit: it's not obvious what the name "origin" means without looking at this script; something like "importDate" or "sourceFileDate" or similar might make it clearer.